rest/route/pod_agent.go

```diff
@@ -120,18 +130,32 @@ func (h *podProvisioningScript) downloadAgentCommands(p *pod.Pod, downloadFromS3
 	)
 	retryArgs := h.curlRetryArgs(curlDefaultNumRetries, curlDefaultMaxSecs)
 
-	var curlCmd string
+	curlExecutable := "curl"
+	if p.TaskContainerCreationOpts.OS == pod.OSWindows {
+		curlExecutable = curlExecutable + ".exe"
```

Comment:
We use this .exe string in several places now, can we make it a global constant?

Reply:
Eh, I don't have strong feelings about this because hosts/pods usually have a lot of plain string manipulation logic when writing shell scripts. I don't think there's a lot of value in making constants out of `.exe` since we don't do it for other common shell script strings (like `bash` and `ssh`) and it's generally understood that `.exe` is a thing for Windows executables.

Reply:
Alright sgtm